internal/output/html/package_table_template.html

```diff
@@ -0,0 +1,38 @@
+<table class="vuln-table">
+  <tr>
+    <th>Package</th>
+    <th>Ecosystem</th>
+    <th>Installed version</th>
+    <th>Fixed version</th>
+    <th>Vulnerability Count</th>
+  </tr>
+  {{ range $rowIndex, $element := . }}
+  {{ if eq (len $element.CalledVulns) 0 }}
+  {{ continue }}
+  {{ end }}
+
+  <tr class='vuln-tr' id="vuln-tr-{{ $element.Name}}-{{ $rowIndex }}"
+    onclick="showVulnDetails('{{ $element.Name }}-{{ $rowIndex }}')">
+    <td>{{ $element.Name }}</td>
+    <td>{{ $element.Ecosystem }}</td>
+    <td>{{ $element.InstalledVersion }}</td>
+    <td {{ if eq $element.FixedVersion "No fix available" }} class="warning" {{ end }}>{{ $element.FixedVersion }}</td>
+    <td>{{ printSeverityCountShort $element.HTMLVulnCount }}</td>
+  </tr>
+  <tr>
+    <td colspan="100%">
+      <div id="vuln-details-{{ $element.Name }}-{{ $rowIndex }}" class="vuln-details">
+        <p><strong>Source: {{ $element.Source }}</strong></p>
+        <p>Vulnerabilities:</p>
+        {{ range $index, $vuln := $element.CalledVulns }}
+          {{ if eq $vuln.Summary.FixedVersion "No fix available" }}
```

Comment:
nit: Keep indentation here on the same level

Reply:
Updated